en/django_views/README.md

```diff
@@ -25,7 +25,7 @@ def post_list(request):
 ```
 
 
-As you can see, we created a method (`def`) called `post_list` that takes `request` and `return` a method `render` that will render (put together) our template `blog/post_list.html`.
+As you can see, we created a function (`def`) called `post_list` that takes `request` and `return` a function `render` that will render (put together) our template `blog/post_list.html`.
```

Comment:
Ah, sorrry! I messed this in my head with the class based views. You are totally right, thank you for correcting me!